app/javascript/tests/subject/monitoring_actions/MonitoringStatus.test.js

```diff
@@ -13,24 +13,31 @@ const authyToken = 'Q1z4yZXLdN+tZod6dBSIlMbZ3yWAUFdY44U06QWffEP76nx1WGMHIz8rYxEU
 const monitoringStatusValues = [ 'Actively Monitoring', 'Not Monitoring' ];
 const monitoringReasons = [
   '--',
+  'Not Monitoring',
```

Comment:
also might be best to move this into its own mock because other test files will need this.  I noticed it was missing from the monitoring actions one so will need to add it there